autoload/committia/git.vim

```diff
@@ -185,6 +201,24 @@ function! committia#git#status() abort
         " Leave status window empty when git-dir or work-tree not found
         return ''
     endtry
+    if g:committia#git#use_verbose || exists('s:use_verbose_status')
+        if exists('s:use_verbose_status') | unlet s:use_verbose_status | end
+        let scissors_line = search('^[#;@!$%^&|:] -\+ >8 -\+\n', 'cenW')
+        if scissors_line > 0
+            " Save cursor position so we can restore it after searching
+            " backwards from the end of the status (i.e. scissors line)
+            " Search backwards to allow for non-default core.commentChar
+            let save_cursor = getcurpos()
+            call cursor(scissors_line, 1)
+            " localisation hack, find the start of the status in the commit
+            " message template using the first line of output from `git status`
+            let status_start = search(split(status, '\n')[0], 'benW')
+            call setpos('.', save_cursor)
```

Comment:
Can we use `getline({lnum} [, {end}])` to search the line instead of `search()` so that we can avoid the side effect by moving the cursor?

Reply:
Makes, sense, will change and test 👍

Reply:
Ah, I'll look into this a little further, I'm already using `getline()` to actually get the relevant line, the reason I'm moving the cursor is to search backwards from the scissors line to find the start of the status. I'll see if I can find a way to do it without moving the cursor.

Reply:
> I'll see if I can find a way to do it without moving the cursor.

I believe it's possible by getting the lines from start to the line by `getline(1, scissors_line)` and iterating them in reversed order.

Reply:
Yeah, could do that, I was going to just search backwards, e.g.

```
            let status_start = search('^[#;@!$%^&|:]' . split(status, '\n')[0], 'benw')
            if status_start > 0 && status_start < scissors_line
```

But let me tinker with both options, iterating in reverse looking for a matching line does sound safer

Reply:
Fix in d726416e, it does seem safer than simply searching backwards (people do strange things, and they should be allowed to)